src/pubtools/_marketplacesvm/tasks/community_push/command.py

```diff
@@ -30,7 +30,6 @@ class CommunityVMPush(MarketplacesVMPush, AwsRHSMClientService):
     """Upload an AMI to S3 and update RHSM."""
 
     _REQUEST_THREADS = int(os.environ.get("COMMUNITY_PUSH_REQUEST_THREADS", "5"))
-    _PROCESS_THREADS = int(os.environ.get("COMMUNITY_PUSH_PROCESS_THREADS", "10"))
```

Comment:
I would keep env variable `COMMUNITY_PUSH_PROCESS_THREADS` for the ThreadPool
with the vale 10. 

So after the new release to Pub we needn't synchronize it removing of `COMMUNITY_PUSH_REQUEST_THREADS=1` from the Pub target. :-)

Reply:
Ah, yes! Ok! I'll adjust it

Reply:
I've changed it back to `COMMUNITY_PUSH_PROCESS_THREADS`, PTAL again